mopidy/models/storage.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import, unicode_literals
+
+import gzip
+import json
+import logging
+import os
+import tempfile
+
+import mopidy
+from mopidy import models
+from mopidy.internal import encoding
+
+logger = logging.getLogger(__name__)
+
+
+def load(path):
+    """
+    Deserialize data from file.
+
+    :param path: full path to import file
+    :type path: str
+    :return: deserialized data
+    :rtype: dict
+    """
+    if not os.path.isfile(path):
+        logger.info('File does not exist: %s.', path)
+        return {}
+    try:
+        with gzip.open(path, 'rb') as fp:
+            return json.load(fp, object_hook=models.model_json_decoder)
+    except (IOError, ValueError) as error:
+        logger.warning(
+            'Loading JSON failed: %s',
+            encoding.locale_decode(error))
+        return {}
+
+
+def save(path, data):
+    """
+    Serialize data to file.
+
+    :param path: full path to export file
+    :type path: str
+    :param data: dictionary containing data to save
+    :type data: dict
+    """
+    data['version'] = mopidy.__version__
```

Comment:
Hmm, adding version here in a general API might be a bit scary, as in we don't know what data already contains. `{'version': mopidy.__version__, 'data': data}` would probably be safer. But it would break how local handles this. So probably easier to make it the callers problem to insert the version number.

That would also make sense as this is currently in a public API, so extension would be allowed to use this. If we want it to be private we should move these helpers to `mopidy.iternal`